R/internal.R

```diff
@@ -41,6 +41,7 @@
 .is_valid_symbolShapes_cxplot <- function(shape) {
     valid_shapes <- .get_valid_symbolShapes_cxplot()
     is_valid_shape <- ifelse(shape %in% valid_shapes, TRUE, FALSE)
```

Comment:
why defning an extra parameter? you can return `ifelse(shape %in% valid_shapes, TRUE, FALSE)` output directly

Reply:
This function will go away - as we are no longer going to support aesthetic related parameters

Reply:
So i will leave everything in internal.R as is for now.